scargo/file_generators/templates/conan/profile_atsam.j2

```diff
@@ -15,7 +15,7 @@ CXX=arm-none-eabi-g++
 [conf]
 tools.build:cflags=["{{ config.project.cflags if config.project.cflags}} {{config.profiles.get(profile).cflags if config.profiles.get(profile).cflags}}"]
 tools.build:cxxflags=["{{ config.project.cxxflags if config.project.cxxflags }} {{ config.profiles.get(profile).cxxflags if config.profiles.get(profile).cxxflags }}"]
-
 {% if config.project.max_build_jobs != None %}
 tools.build:jobs={{config.project.max_build_jobs}}
 {% endif %}
+tools.cmake.cmaketoolchain:user_toolchain=["$ENV{SCARGO_PROJECT_ROOT}/config/conan/toolchain/arm_gcc_toolchain.cmake"]
```

Comment:
The profile should be machine independent - otherwise the packages will be rebuilt on each machine separately.
Instead I suggest to use variable `PROFILE_DIR`
e.g.,
`$PROFILE_DIR/arm-gcc-toolchain.cmake`

Reply:
Sure, it makes it quite confusing in our jinja file since in conan2 profiles are rendered using jinja2, but it does work.
https://docs.conan.io/2/reference/config_files/profiles.html?highlight=profile_dir#profile-rendering